core/banking-protocol/src/main/java/de/adorsys/opba/core/protocol/controller/BankController.java

```diff
@@ -25,6 +26,7 @@
     description = "Bank search API",
     externalDocs = @ExternalDocumentation(description = "Search bank and select bank profile")
 )
+@CrossOrigin(origins = "*")
```

Comment:
It is better to control CORS at gateway/load balancer level (i.e. in HAProxy etc.)

Reply:
I don't see how we can integrate with frontend now. Maybe we can change it later?

Reply:
In such case create ticket for that and reference it under FIXME comment

Reply:
done